### ISSUE TITLE
PWA support is available?

### ISSUE THREAD

**Issue by MarceloRosendo** [olefirenko/vue-barcode-reader#12]
On computer web browser the example works effectivelly, but on mobile chome browser is not working.

**Comment by Cristian-Ayala** [comment]
Sure, I'm developing a PWA, and i't working for me. You can download the files needed and add them into your caching strategies.

**Comment by PMLS3** [comment]
is the stream version working for pdf417 on pwa? Image uploads works but the scanner (IOS) not working for me

**Comment by olefirenko** [comment]
Yes, it works fine in PWAs. You can check here as an example https://parceltrackingapp.com/en
